server/models/video/video-abuse.ts

```diff
@@ -408,14 +449,31 @@ export class VideoAbuseModel extends Model<VideoAbuseModel> {
   }
 
   toActivityPubObject (this: MVideoAbuseVideo): VideoAbuseObject {
+    const predefinedReasons = VideoAbuseModel.getPredefinedReasonsStrings(this.predefinedReasons || [])
```

Comment:
Put the `null` check inside `getPredefinedReasonsStrings` so you don't have to use `|| []` anymore